src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/response/model/SentenceDetail.kt

```diff
@@ -1,55 +1,63 @@
 package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.response.model
 
+import jakarta.validation.constraints.NotNull
+import java.time.LocalDate
+
 class SentenceDetail {
```

Comment:
Same as the last comment. This is a huge class whose details we are very unlikely to need more than a few fields from.